doc/3/controllers/collection/search-specifications/index.md

```diff
@@ -1,6 +1,7 @@
 ---
 code: true
 type: page
+<<<<<<< HEAD
```

Comment:
Merge leftover

Reply:
I think you are not up to date, check the last commit